.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -119,18 +119,22 @@ jobs:
           mkdir Kokkos
           cp -rf /root/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
-      - name: Install dependencies (CentOS)
-        if: ${{ (matrix.container_img == 'quay.io/pypa/manylinux2014_x86_64') }}
+      - name: Install dependencies (AlmaLinux)
+        if: ${{ (matrix.container_img == 'quay.io/pypa/manylinux_2_28_x86_64') }}
         run: |
-          # Reduce wait time for repos not responding
-          cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
-          yum update -y && yum install -y docker
+          yum update -y
+          yum install yum-utils -y
+          yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+          yum install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
```

Comment:
:thinking: Wouldn't the GitHub matrix strategy take care of the container engine?

I mean this line,
```
        container_img: ["quay.io/pypa/manylinux_2_28_x86_64"]
```

Reply:
cibuildwheels create a container inside the container. This is for the second layer of containerization.
As I said before, I'm just exploring the problem.

Reply:
Thanks for exploring! I was asking for my own understanding :)